.eslintrc.js

```diff
@@ -1,4 +1,53 @@
 module.exports = {
   root: true,
-  extends: '@react-native-community',
-};
+  parser: 'babel-eslint',
+  parserOptions: {
+    ecmaVersion: 9,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true
+    }
+  },
+  plugins: ['react', 'flowtype'],
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'standard',
+    'plugin:jest/recommended'
+  ],
+  env: {
+    browser: true,
+    node: true,
+    es6: true
+  },
+  settings: {
+    react: {
+      createClass: 'createReactClass', // Regex for Component Factory to use,
+      // default to "createReactClass"
+      pragma: 'React', // Pragma to use, default to "React"
+      version: 'detect', // React version. "detect" automatically picks the version you have installed.
+      // You can also use `16.0`, `16.3`, etc, if you want to override the detected value.
+      flowVersion: '0.155.1' // Flow version
```

Comment:
versi flownya dinaikkan ya?
```suggestion
      flowVersion: '0.92.0' // Flow version
```